### DISCUSSION TITLE
Project restoration takes greater than 5 hours?

### DISCUSSION THREAD

**Discussion by ibsusu** [supabase/supabase#19946] [Questions] [answered]
Good day, My database is pretty tiny, less than 300KB easy.  I'm wondering if taking more than 5ish hours to restore a project is normal.  I found another discussion saying that I should contact support for this specific issue.  I have done that, but I got an email saying that I should come here for support... so I'm kinda stuck.

**Comment by GaryAustin1** [comment, marked as answer]
I assume the email said you could also come here.
Normally a small database should be minutes to restore.  
Only support can get your instance restored when it is in this state.

**Reply by ibsusu** [reply]
I've just upgraded to pro.  1 business day is slow but hopefully help comes along sooner.  Thank you.

**Reply by GaryAustin1** [reply]
I assume you tried hard refreshing the browser and a different browser just to make sure it is not just lost in the UI.

**Reply by ibsusu** [reply]
correct, soft refreshing, hard refreshing, safari, firefox, logging in and out.  the error:
<img width="356" alt="Screenshot 2023-12-21 at 10 44 30 AM" src="https://github.com/supabase/supabase/assets/3039453/cf470a26-9729-4bda-a281-295444587680">

**Comment by ibsusu** [comment]
in this case support did help.
